ccf_transaction_processor/transaction_processor/pdo_tp.h

```diff
@@ -67,18 +81,27 @@ namespace ccfapp
     {
         private:
 
+            kv::Map<string, EnclaveExpectedMeasurements> enclave_expected_measurements; 
+                        // There is a single entry with key PDO_ENCLAVE_EXPECTED.
+                        // Only the CCF Governing body can update this entry.
+                        // Can be generalized if multiple enclave "types" need to be verified.
             kv::Map<string, EnclaveInfo> enclavetable; // key is encalve_id
             kv::Map<string, ContractInfo> contracttable; // key is contract_id
             kv::Map<string, ContractStateInfo> ccltable; // key is contract_id + state_hash (string addition)
             kv::Map<string, map<string, string>> signer; //There is at most one entry in this map. if there is an
-                                                            //entry key="signer".  value is pubk:privk
+                                                         //entry key="signer".  value is pubk:privk
 
             // functions to verify signatures, only wite methods sign transactions, read methods do not.
             bool verify_pdo_transaction_signature_register_enclave(
                 const vector<uint8_t>& signature,
                 const string & verifying_key,
                 const EnclaveInfo & enclave_info);
 
+            bool verify_rsa_sig(
```

Comment:
While the name is obvious, it's helpful to disambiguate, especially considering the `verify_sig` function below:
```suggestion
             /* RSA sig verification */
            bool verify_rsa_sig(
```

Reply:
thank you, comment added.